combinedGame.js

```diff
@@ -0,0 +1,132 @@
+    let playerNumber = await ask('Now pick a number for me to guess!');
+    //return the chosen number to the player for confirmation 
+    console.log(`you've picked ${playerNumber}, now i'll try and guess it!`)
+    //establish the initial guess by the computer 
+    let pcGuess = randomNum(playerMin, playerMax)
+    //variable to keep track of guesses
+    let guessNum = 0
+    //establish a loop to allow the game to function without a million else if functions
+    while (true) {
+        //increase the value of the guess tracking variable by one each loop
+        ++guessNum
+        //guess a random number and ask if its correct
+        let guess1 = await ask(`${pcGuess} , is this your number? [y]es or [n]o`)
+        console.log(pcGuess)
+        //if the guess is correct gloat and end the game
+        if (guess1 === 'y') {
+            console.log('Ha,I win! Try harder next time!')
+            //print guess number
+            console.log(`It only took me ${guessNum} tries too!`)
+            //code to restart function if you want to play again
+            let playAgain = await ask('Would you like to play again? [y]es or [n]o')
+            if (playAgain === "y") {
+                pickGame()
+            } else if (playAgain === "n") {
+                console.log('Goodbye then!')
+                process.exit()
+
+            }
+            //if the guess is not correct ask if the number was higher or lower and guess again with an updated range 
+        } else if (guess1 === "n") {
+            let guess2 = await ask('is your number [higher] or [lower] than mine?')
+            //if the player lies end the game 
+            if (guess2 === 'higher' && pcGuess > playerNumber) {
+                console.log("I don't play with cheaters")
+                process.exit()
+            } else if (guess2 === 'higher') {
+                //change the range and guess "smart" by guessing the middle of the new range 
+                //hard change to value of parsedPlayerMin by one to prevent guessing the same number 
+                parsedPlayerMin = pcGuess + 1
+                pcGuess = smartNum(parsedPlayerMin, parsedPlayerMax)
+                //guard against the player lying
+            } else if (guess2 === 'lower' && pcGuess < playerNumber) {
+                console.log("I don't play with cheaters")
+                process.exit()
+            } else if (guess2 === 'lower') {
+                parsedPlayerMax = pcGuess - 1
+                pcGuess = smartNum(parsedPlayerMin, parsedPlayerMax)
+            }
+        }
+    }
+}
+//simple ask prompt to decide which function to run. run one or the other function depending on the answer
+async function pickGame() {
+    let promt = await ask(`Let's try to guess some numbers!
+    Would you like me to pick a number, or do you want to pick a number?`)
+    if (promt === 'you pick') {
+        guessReverse()
+    } else if (promt === "I pick") {
+        guessGame()
+    }
```

Comment:
I had to look into the code to find out what inputs you were expecting in order to play the game, or else it just hung there and I had to restart that game. This would be a good place for a sanitize function or be more specific in your user requests(ie. `Let's try to guess some numbers! Would you like me to pick a number[1], or do you want to pick a number[2]?`)